bip-0340.mediawiki

```diff
@@ -97,6 +97,8 @@ This proposal suggests to include the tag by prefixing the hashed data with ''SH
 
 '''Final scheme''' As a result, our final scheme ends up using public key ''pk'' which is the X coordinate of a point ''P'' on the curve whose Y coordinate is even and signatures ''(r,s)'' where ''r'' is the X coordinate of a point ''R'' whose Y coordinate is a square. The signature satisfies ''s⋅G = R + tagged_hash(r || pk || m)⋅P''.
 
+We note that adapting this specification to other elliptic curves is not straightforward and can result in an insecure scheme<ref>Among other pitfalls, using the specification with a curve whose order is not close to the output of the nonce derivation is insecure.</ref>.
```

Comment:
```suggestion
We note that adapting this specification to other elliptic curves is not straightforward and can result in an insecure scheme<ref>Among other pitfalls, using the specification with a curve whose order is not close to the size of the range of the nonce derivation function is insecure.</ref>.
```

I'd move it somewhere to the top of  the Specification section to make it more prominent. I guess it fits after the introductory sentence there.

Reply:
Fixed your nit. I had the sentence at the top of the Spec section before, but the Design section seems slightly more fitting.

Reply:
Do you think it fits better there? I believe the opposite is true, he sentence specifically talks about the specification. (No pun intended.) I somehow fear people could miss it because they simply ignore the Design section. But I guess for every position in the BIP, you'll find some person who'll ignore it.

Reply:
moved it to the spec